### ISSUE TITLE
training custom model, is this accuracy reasonable?

### ISSUE THREAD

**Issue by mugenZebra** [tsurumeso/waifu2x-chainer#30]
`
!python /content/waifu2x-chainer/train.py --gpu 0 --dataset_dir ./jpg --patches 32 --epoch 10 --model_name reference_scale_rgb --downsampling_filters box lanczos --lr_decay_interval 3 --arch UpConv7
/usr/local/lib/python3.6/dist-packages/chainer/_environment_check.py:91: UserWarning: 
--------------------------------------------------------------------------------
Multiple installations of CuPy package has been detected.
You should select only one package from from ['cupy-cuda102', 'cupy-cuda101', 'cupy-cuda100', 'cupy-cuda92', 'cupy-cuda91', 'cupy-cuda90', 'cupy-cuda80', 'cupy'].
Follow these steps to resolve this issue:
  1. `pip list` to list CuPy packages installed
  2. `pip uninstall <package name>` to uninstall all CuPy packages
  3. `pip install <package name>` to install the proper one
--------------------------------------------------------------------------------

  '''.format(name=name, pkgs=pkgs))
* loading filelist... done
* setup model... done
* check forward path... done
* starting processes of dataset sampler... done
### epoch: 0 ###
  # inner epoch: 0
    * best loss on training dataset: 0.002381
    * best score on validation dataset: PSNR 22.072546 dB
    * elapsed time: 881.055754 sec
  # inner epoch: 1
    * best loss on training dataset: 0.001941
    * best score on validation dataset: PSNR 22.579106 dB
    * elapsed time: 103.186740 sec
  # inner epoch: 2
    * best loss on training dataset: 0.001810
    * best score on validation dataset: PSNR 22.679653 dB
    * elapsed time: 97.545877 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001732
    * best score on validation dataset: PSNR 23.013222 dB
    * elapsed time: 91.364418 sec
### epoch: 1 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.169959 dB
    * elapsed time: 106.030671 sec
  # inner epoch: 1
    * best loss on training dataset: 0.001652
    * best score on validation dataset: PSNR 23.181128 dB
    * elapsed time: 102.384226 sec
  # inner epoch: 2
    * best loss on training dataset: 0.001593
    * best score on validation dataset: PSNR 23.337347 dB
    * elapsed time: 99.998622 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001545
    * elapsed time: 91.267800 sec
### epoch: 2 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.427455 dB
    * elapsed time: 106.444332 sec
  # inner epoch: 1
    * elapsed time: 103.891084 sec
  # inner epoch: 2
    * best loss on training dataset: 0.001530
    * elapsed time: 98.481055 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001489
    * best score on validation dataset: PSNR 23.479888 dB
    * elapsed time: 91.023230 sec
### epoch: 3 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.528935 dB
    * elapsed time: 106.155661 sec
  # inner epoch: 1
    * elapsed time: 103.248413 sec
  # inner epoch: 2
    * best loss on training dataset: 0.001455
    * elapsed time: 98.362952 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001421
    * learning rate decay: 0.000225
    * elapsed time: 90.741462 sec
### epoch: 4 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.533943 dB
    * elapsed time: 105.805502 sec
  # inner epoch: 1
    * best score on validation dataset: PSNR 23.573726 dB
    * elapsed time: 103.978917 sec
  # inner epoch: 2
    * best score on validation dataset: PSNR 23.623575 dB
    * elapsed time: 99.085195 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001405
    * best score on validation dataset: PSNR 23.658209 dB
    * elapsed time: 91.353107 sec
### epoch: 5 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.710338 dB
    * elapsed time: 106.360211 sec
  # inner epoch: 1
    * best score on validation dataset: PSNR 23.714681 dB
    * elapsed time: 104.392316 sec
  # inner epoch: 2
    * elapsed time: 99.168070 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001383
    * elapsed time: 91.488422 sec
### epoch: 6 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.732231 dB
    * elapsed time: 107.400784 sec
  # inner epoch: 1
    * best score on validation dataset: PSNR 23.761270 dB
    * elapsed time: 105.255023 sec
  # inner epoch: 2
    * elapsed time: 99.553206 sec
  # inner epoch: 3
    * elapsed time: 91.538299 sec
### epoch: 7 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.764517 dB
    * elapsed time: 106.978499 sec
  # inner epoch: 1
    * elapsed time: 104.257074 sec
  # inner epoch: 2
    * elapsed time: 99.090099 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001361
    * learning rate decay: 0.000203
    * elapsed time: 91.054147 sec
### epoch: 8 ###
  # inner epoch: 0
    * best score on validation dataset: PSNR 23.816657 dB
    * elapsed time: 105.482888 sec
  # inner epoch: 1
    * best score on validation dataset: PSNR 23.883949 dB
    * elapsed time: 104.894173 sec
  # inner epoch: 2
    * elapsed time: 99.114435 sec
  # inner epoch: 3
    * elapsed time: 90.749769 sec
### epoch: 9 ###
  # inner epoch: 0
    * learning rate decay: 0.000182
    * elapsed time: 92.891063 sec
  # inner epoch: 1
    * elapsed time: 90.760490 sec
  # inner epoch: 2
    * elapsed time: 90.826175 sec
  # inner epoch: 3
    * best loss on training dataset: 0.001345
    * learning rate decay: 0.000164
    * elapsed time: 90.922554 sec`

I read that the PSNR for the pretrained model is 30+, is 23dB too low? Maybe my training set is bad? My training set consists of mostly black and white images, specifically Japanese comic book pages, if that matters.
